src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Generator.cs

```diff
@@ -230,7 +230,6 @@ public GenerationResult GenerateRootPackage(IInternalMetadataProvider internalMe
         };
 
         spdxSupplier.AddSpdxId();
-        spdxPackage.AddSpdxId();
```

Comment:
Not sure why this is removed--was it duplicated somewhere else?

Reply:
For root packages, we set the SPDX ID differently to indicate it's a root package. I do set the SPDX ID initially using a constant value on line 205. This was an unintentional duplicate that overwrote the constant SPDX ID.